SCClassLibrary/Common/Audio/SynthDef.sc

```diff
@@ -541,9 +541,10 @@ SynthDef {
 	}
 
 	// make SynthDef available to all servers
-	add { | libname = \allRunningServers, completionMsg, keepDef = true |
+	add { | libname = \global, completionMsg, keepDef = true |
 		var servers;
-		if(libname.isNil || (libname == \allRunningServers)) {
+		this.asSynthDesc(libname ? \global, keepDef);
+		if(libname.isNil || (libname == \global)) {
```

Comment:
Since `\global` is the actual name of a library, I think this check should only be for `.isNil`. Users can add their own servers to `SynthDescLib`s, so someone may get `allRunningServers` instead of `SynthDescLib.global.servers`.